lbr_python_tools/lbr_python_tools/lbr_node.py

```diff
@@ -0,0 +1,158 @@
+import os
```

Comment:
could you please give me an explanation for this package so I can understand it's purpose better?